City-of-Helsinki/drupal-module-helfi-tpr
# UHF-8730 tpr unit new field tests

[UHF-8730](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8730)

## What was done?
* Added tests for the new connection fields.

## How to install

* Make sure your instance is up and running on correct branch.
  * `make fresh`
* Run `composer require drupal/helfi_platform_config:dev-UHF-8730_Unit-new-field-tests`

## How to test


Run following in the `/app` inside the container.
* [ ] Run `vendor/bin/phpunit -c /app/phpunit.xml.dist public/modules/contrib/helfi_tpr/tests/src/Unit/ConnectionRepositoryTest.php`
* [ ] Run `vendor/bin/phpunit -c /app/phpunit.xml.dist public/modules/contrib/helfi_tpr/tests/src/Unit/ConnectionTest.php`
* [ ] Run `vendor/bin/phpunit -c /app/phpunit.xml.dist public/modules/contrib/helfi_tpr/tests/src/Kernel/UnitMigrationTest.php`

All tests should pass.

[UHF-8730]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-8730?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
